### PR TITLE
feat(multiarch): Adding multiarch support

### PR DESCRIPTION
**Pull request by ecktom**

### CODE REVIEW

File: .buildkite/pipeline.sh [hunk 1]
```diff
@@ -191,7 +191,7 @@ main() {
                     --arg image_name "${IMAGE_NAME}" \
                     --argjson step_env "${STEP_ENV}" \
                     --slurpfile steps "${STEPS_DIR}/${STEP}.json" \
-                    'reduce $steps[] as $step (.; .steps[.steps | length] = ($step | map(.env=$step_env) | map(.env.IMAGE_NAME=$image_name | .name=$friendly_name + " " + .name))) | .steps = (.steps | flatten)' \
+                    'reduce $steps[] as $step (.; .steps[.steps | length] = ($step | map(.env=.env+$step_env) | map(.env.IMAGE_NAME=$image_name | .name=$friendly_name + " " + .name))) | .steps = (.steps | flatten)' \
```
Comment (tobjaw) [inline .buildkite/pipeline.sh:194]: I see you like playing with fire 😄

**Review by tobjaw — APPROVED** [review]
Good stuff!

**Comment by jhedev** [comment]
Do we need to create the `gridx/golang-tools-arm` repo before merging?

**Review by jhedev — APPROVED** [review]
:+1: :D

**Pull request merged**